examples/musig.c

```diff
@@ -0,0 +1,218 @@
+/*************************************************************************
+ * Written in 2018 by Jonas Nick                                         *
```

Comment:
(Okay, I'm all in favor of not doing updating years and stuff, but this is just too wrong. :D) 
```suggestion
 * Written by Jonas Nick                                                 *
```

Reply:
Removed name and year from all headers